app/(app)/settings/language.tsx

```diff
@@ -44,26 +50,24 @@ const Page = () => {
   return (
     <ScreenView title={t('title')} hasBackButton>
       <ScreenContent>
-        <View>
-          <PressableStyled onPress={setSystemLanguage} className="py-4">
-            <FlexRow>
-              <Typography>{t('system')}</Typography>
-              {/* TODO fix check icon */}
-              {mmkvLocale === undefined && <Icon name="check-circle" />}
-            </FlexRow>
-          </PressableStyled>
+        <View className="divide-y divide-divider">
```

Comment:
Please doublecheck the design when changing something like this. Figma specifies 2px divider.

Reply:
![Screenshot 2024-01-12 at 12 26 49](https://github.com/bratislava/paas-mpa/assets/61841194/63c4ab1d-b7a0-4e90-96d5-3ff4db05db25)